sync/sync-impl/src/main/java/com/duckduckgo/sync/impl/ui/setup/SetupAccountActivity.kt

```diff
@@ -55,24 +50,12 @@ class SetupAccountActivity : DuckDuckGoActivity(), SetupFlowListener {
 
     private lateinit var screen: Screen
 
-    private val loginFlow = registerForActivityResult(LoginContract()) { resultOk ->
-        if (resultOk) {
-            viewModel.onLoginSucess()
-        }
-    }
-
-    private val connectFlow = registerForActivityResult(ConnectFlowContract()) { resultOk ->
-        if (resultOk) {
-            viewModel.onLoginSucess()
-        }
-    }
-
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
-        if (savedInstanceState != null) {
-            screen = savedInstanceState.getSerializable(SETUP_ACCOUNT_SCREEN_EXTRA) as Screen
+        screen = if (savedInstanceState != null) {
+            savedInstanceState.getSerializable(SETUP_ACCOUNT_SCREEN_EXTRA) as Screen
         } else {
-            screen = intent.getSerializableExtra(SETUP_ACCOUNT_SCREEN_EXTRA) as? Screen ?: SETUP
+            intent.getSerializableExtra(SETUP_ACCOUNT_SCREEN_EXTRA) as? Screen ?: DEVICE_SYNCED
```

Comment:
Previously, default value was SETUP (new: sync_setup). Have we changed the default value for anything in particular? (I would expect if no value indicated, correct value is sync_setup for this flow)

Which brings me the question, should we automatically close activity if user enters with sync_setup but signedin...could be something quick to cover this case.

Reply:
Discussed offline, we’ll stick with this.